src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -368,29 +377,60 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Character) + td:nth-child(2) > a")
+                .joinToString { it.text() }
+        }
+        val SeriesName = when {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr.parodies > td:nth-child(2) > a")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select("section.metadata > table:nth-child(2) > tbody > tr.parodies > td:nth-child(2) > a")
+                .joinToString { it.text() }
+        }
+        val SeriesParser = if ("Manhwa" in SeriesName) {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr:nth-child(5) > td:nth-child(2)").text()
+        } else {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr.parodies > td:nth-child(2) > a").text()
+        }
+        val AlternativeTitle = when {
+            document.select("section.metadata > h1.title > span.alter")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select("section.metadata > h1.title > span.alter")
+                .joinToString { it.text() }
+        }
         val manga = SManga.create()
         manga.description = when {
             document.select("section.metadata > div.pb-2 > p:nth-child(1)")
-                .isEmpty() -> "Tidak ada deskripsi yang tersedia bosque"
-            else -> document.select("section.metadata > div.pb-2 > p:nth-child(1)").first()!!.text()
+                .isEmpty() -> "Tidak ada deskripsi yang tersedia bosque\n\nJudul Alternatif : $AlternativeTitle"
+            else -> (document.select("section.metadata > div.pb-2 > p:nth-child(1)").first()!!.text() + "\n\nJudul Alternatif : $AlternativeTitle")
         }
         val genres = mutableListOf<String>()
         infoElement.select("div.tags > a").forEach { element ->
             val genre = element.text()
             genres.add(genre)
         }
-        manga.author =
-            document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Author) + td:nth-child(2) > a")
-                .joinToString { it.text() }
+        manga.author = "Author : $AuthorName\nGroup  : $GroupName"
         manga.genre = infoElement.select("div.tags > a").joinToString { it.text() }
         manga.status = parseStatus(
             document.select("section.metadata > table:nth-child(2) > tbody > tr:nth-child(1) > td:nth-child(2) > a")
                 .first()!!.text(),
         )
         manga.thumbnail_url = document.selectFirst("figure.thumbnail img")?.attr("src")
-        manga.artist =
-            document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Character) + td:nth-child(2) > a")
-                .joinToString { it.text() }
+        manga.artist = "Character : $CharacterName\nSeries       : $SeriesParser"
```

Comment:
Belongs in the description.

Reply:
Got it, soo manga.artist not used?

Reply:
`manga.artist` should be used for the artist. If the information is not provided by the site it should be omitted.

Reply:
But in DouDesu didn't have an artist Section, they only have author, group, series, rating, day created, status, and type